src/lib/Map.svelte

```diff
@@ -17,11 +17,23 @@
 	let mapElem = $state<HTMLDivElement>();
 	const vending = writable<string>('');
 	const payment = writable<string>('');
-	const vendings = [
-		{ value: 'drinks', title: '飲み物' },
-		{ value: 'bread', title: 'パン' },
-		{ value: 'ice_cream', title: 'アイス' }
-	];
+	const VENDING = {
+		DRINKS: {
+			value: 'drinks',
+			title: '飲み物',
+			icon: { id: 'icon-bottle', file: 'icon-bottle.webp' }
+		},
+		BREAD: {
+			value: 'bread',
+			title: 'パン',
+			icon: { id: 'icon-bread', file: 'icon-baguette.webp' }
+		},
+		ICE_CREAM: {
+			value: 'ice_cream',
+			title: 'アイス',
+			icon: { id: 'icon-icecream', file: 'icon-icecream.webp' }
+		}
```

Comment:
iconの色もobjectに含めてもよさそうな気がしました
```suggestion
		DRINKS: {
			value: 'drinks',
			title: '飲み物',
			icon: { id: 'icon-bottle', file: 'icon-bottle.webp', color: 'blue' }
		},
		BREAD: {
			value: 'bread',
			title: 'パン',
			icon: { id: 'icon-bread', file: 'icon-baguette.webp', color: 'yellow' }
		},
		ICE_CREAM: {
			value: 'ice_cream',
			title: 'アイス',
			icon: { id: 'icon-icecream', file: 'icon-icecream.webp', color: 'red' }
		}
```